tests/pytests/test_certifier_framework.py

```diff
@@ -604,6 +604,32 @@ def test_run_app_as_a_server_with_trust_manager():
     assert result is True
     print(' ... cfm.server_dispatch() with cc_trust_manager &mgr interface succeeded.')
 
+# ##############################################################################
+@pytest.mark.needs_cert_service()
+@pytest.mark.check_leaks()
+def test_trust_manager_write_private_key_to_file():
```

Comment:
Unit-Test case written to exercise and test this new interface ... that it's basically working.